City-of-Pori/pori-kada
# PORI-292 Added configuration that fixes google analytics id based on …

…domain, fixed feature conflicts

To test:

1. drush fra -y && drush cc all
2. Go to check https://local.pori.fi/en/admin/structure/domain/variables that the "Web Property ID" is checked.
3. Go to check https://local.pori.fi/en/admin/config/system/googleanalytics?variable_realm_key_domain=visitpori_fi that the visitpori "Web Property ID" is UA-76043723-3 and the pori one is UA-76043723-2
